File: interface/data/accounts/CreateAccountOptions.ts
// Copyright © Spatial Corporation. All rights reserved.

import { CreateResourceOptions } from "..";

/**
 * Configurable options for a new account.
 */
export type CreateAccountOptions = CreateResourceOptions & {
  /**
   * The user's name.
   */
  name: string;

  /**
   * The user's email address.
   */
  email: string;

  /**
   * Configurable options for the account.
   */
  metadata?: Record<string, string>;
};

File: cloud/Server/Data/Accounts/CreateAccountOptions.cs
// Copyright © Spatial Corporation. All rights reserved.

namespace Spatial.Cloud.Data.Accounts;

/// <summary>
/// Configurable options for a new account.
/// </summary>
public class CreateAccountOptions : CreateResourceOptions
{
    /// <summary>
    /// The user's name.
    /// </summary>
    public string Name { get; set; }

    /// <summary>
    /// The user's email address.
    /// </summary>
    public string Email { get; set; }
}